src/test/lib/storage/index/adaptive_radix_tree/adaptive_radix_tree_index_test.cpp

```diff
@@ -44,28 +44,29 @@ class AdaptiveRadixTreeIndexTest : public BaseTest {
       pairs.emplace_back(std::make_pair(bc, values1[i]));
     }
     root = index1->_bulk_insert(pairs);
-
-    std::random_device rd;
-    _rng = std::mt19937(rd());
   }
 
   void _search_elements(std::vector<std::optional<int32_t>>& values) {
-    std::uniform_int_distribution<int32_t> uni_integer(0, std::numeric_limits<int32_t>::max());
-
     auto segment = create_dict_segment_by_type<int32_t>(DataType::Int, values);
     auto index =
         std::make_shared<AdaptiveRadixTreeIndex>(std::vector<std::shared_ptr<const AbstractSegment>>({segment}));
 
     std::set<std::optional<int32_t>> distinct_values(values.begin(), values.end());
 
-    // create search values from given values + randomly chosen ones
+    // 0, 1, the maximum int32_t value and 20 randomly generated positive int32_t values
+    const auto additional_value_pool = std::array<int32_t, 23>{
+        0,          1,          2147483647, 359982793,  2079147571, 386510393,  1762525179, 664094870,
+        1937011651, 784616030,  1000301793, 1891243447, 1654409109, 1476995155, 1110747240, 1726885375,
+        400463838,  2100392285, 308227485,  616812231,  997147072,  2121290667, 986682158};
```

Comment:
Good solution. Why only positive values? I feel like negative values are underrepresented in our tests. Same for the EncodedSegmentTest.

Reply:
I used the interval that was used before. But of course, we can extend it.